dotVimRc

```diff
@@ -89,7 +89,12 @@ set wildignore+=*/node_modules/**
 " Including this messes up fugitive in git mergetool:
 " set wildignore+=*/.git/*,*/.hg/*,*/.svn/*        " Linux/MacOSX
 set sm!  " show matching brace/paren
+
+" Disable visualbell by default
+" http://unix.stackexchange.com/a/5313
```

Comment:
One commend says it must also be set in .gvimrc. Did you find that to be correct?